Noto/CDB/alma/RECEIVERS/NotoReceivers/NotoReceivers.xml

```diff
@@ -10,7 +11,7 @@
 				xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"   
 				HPIBIPAddress="192.167.187.201"
 				HPIBPort="1234"
-				LocalOscillatorInstance="RECEIVERS/LocalOscillator"
+				LocalOscillatorInstance=""
 				LocalOscillatorEnabledReceivers="KKC QQC CCC MMC"				
```

Comment:
I'm not sure about this modification, with no instance of LO we still tell the system that KKC QQC CCC and MMC receivers can use the local oscillator?

Reply:
I just noticed this is the test CDB, the question still remains, though

Reply:
Just checked: this modification is also on the production Database. In any case this is temporary. The control of the local oscillator is not working at the moment and I do not what the receiver configuration is blocked because of that. When the problem is solved this configuration will be reverted.

Reply:
@giuseppe-carboni I addressed your comments, also other two commits with small configuration fixes have been done